react-forms/src/components/RegForm/RegForm.tsx

```diff
@@ -0,0 +1,201 @@
+import { ChangeEvent, FormEvent, useState } from 'react'
+import s from './RegForm.module.scss'
+import classnames from '../../utils/classnames'
+import Form, { OneLine } from '../@common/@form/Form/Form'
```

Comment:
Won't check the content of `@common` ... I'll assume that content has been crafted properly and has no errors.